WalletWasabi.Fluent/ViewModels/Wallets/WalletSettingsViewModel.cs

```diff
@@ -20,6 +24,7 @@ public WalletSettingsViewModel(WalletViewModelBase walletViewModelBase)
 		_autoCoinJoin = wallet.KeyManager.AutoCoinJoin;
 		IsHardwareWallet = wallet.KeyManager.IsHardwareWallet;
 		IsWatchOnly = wallet.KeyManager.IsWatchOnly;
+		_plebStopThreshold = wallet.KeyManager.PlebStopThreshold.ToString();
```

Comment:
Should be initialized when the navigation happened. The wallet settings is created only once and only navigation happens to that.

If the user typed something incorrect (i.e.: a text) into the field, then leaves the page, then open it again. The incorrect text would be visible in an error state.

Can you fix it?

Reply:
Done.
Small thing, i.e. if the user enters `10`, then exits and returns, it will be `10.00000000' precisely.
Is this a bad UX? If the user didn't enter a fraction, should we show an integer?

Reply:
It's fine IMO